xml/System.Numerics/BitOperations.xml

```diff
@@ -146,8 +146,8 @@
         <Parameter Name="value" Type="System.UInt64" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="value">The value.</param>
+        <summary>Evaluate whether a given integral value is a power of 2.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <param name="value">The value to evaluate as being a power of two.</param>
        <summary>Evaluates whether the specified <see cref="T:System.UInt64" /> value is a power of two.</summary>
        <returns><see langword="true" /> if the specified value is a power of two; <see langword="false" /> otherwise.</returns>
```